### DISCUSSION TITLE
[QUESTION] How do I change the logo and add links to the homepage?

### DISCUSSION THREAD

**Discussion by moeraccoon** [searxng/searxng#2380] [Q&A] [answered]
I have a picture I would like to use as the logo in the homepage, replacing the searxng text.
I also saw on another instance that under the search bar they added icons for each other service the website provides and I wanted to add that as well. How can I do this?

I searched and yes, there are similar questions, but I couldn't quite well understand because I'm still very new and learning, if anyone could help me out with a simple explanation I would be very grateful.

**Comment by return42** [comment, marked as answer]
duplicate: https://github.com/searxng/searxng/discussions/2032
